packages/design-system/src/theme/colors.ts

```diff
@@ -103,4 +117,18 @@ export const colors: MetaColors = {
     800: '#0d1a40',
     900: '#02091b',
   },
+  landing50: '#1A0340',
```

Comment:
And replace this with
```
landingPurple: {
50: '#1A0340',
100: '...',
...
..
900: '...',
}

Reply:
I did start like this but got unexpected results but will re-try it. 🙏🏻